elide-contrib/elide-dynamic-config-helpers/src/main/resources/templates/table.hbs

```diff
@@ -87,6 +93,9 @@ public class <%#capitalizeFirstLetter name%><%/capitalizeFirstLetter%> <%#if ext
     <%#if readAccess%>@ReadPermission(expression = "<%readAccess%>")<%/if%>
     <%#if category%>@Meta(description = "<%description%>", category="<%category%>")<%else%>@Meta(description = "<%description%>")<%/if%>
     <%#if hidden%>@Exclude<%/if%>
+    <%#if override%>
+    @Getter(onMethod = @__({@Override})) @Setter(onMethod = @__({@Override}))
```

Comment:
Lets put @setter on a new line. Also, any reason for using 2 different type of annotations? 
`@Getter(onMethod = @__({@Override}))` and `@Getter(onMethod_={@Override})`